.github/ISSUE_TEMPLATE.md

```diff
@@ -2,6 +2,7 @@
 
 - Cloud Tools for Eclipse version:
 - Google Cloud SDK version: _(run `gcloud version`)_
+- Eclipse version: _(e.g, 4.5.2.v20160212-1500 or Neon.3)_
```

Comment:
if we're autofilling this, no need to add the e.g....

Reply:
This is the template used when users open an issue on GitHub directly, not through our menu.